src/app/file-explorer.js

```diff
@@ -20,12 +20,9 @@ var css = csjs`
     background-color  : white;
   }
   .remove             {
-    align-self        : center;
-    padding-left      : 10px;
+    float             : right;
```

Comment:
using `align-self: center` is not really appropriate cause we don't want to center it but to push it on the right..
yes it was introduce by this commit, but the commit should stay i think.
about using `flex`, as I understood it, this is a good way to distribute space between component in various ways, but does it apply to our case? cause here we just want to push the remove icon close to the right border